content/v2.0/reference/flux/stdlib/contrib/teams/message.md

```diff
@@ -0,0 +1,78 @@
+---
+title: teams.message() function
+description: >
+  The `teams.message()` function sends a single message to a Microsoft Teams channel using
+  a [incoming webhook](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks&amp?page=3).
```

Comment:
should this be URL on line 5?:
https://docs.microsoft.com/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

Reply:
Yes, that way it shows up with this doc appears in the a list generated by the children shortcode. The link html tag gets stripped out when it builds the page meta description.